products/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from django.http      import JsonResponse
+from django.views     import View
 
-# Create your views here.
+from products.models  import Product
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        try:
+            product            = Product.objects.get(id = product_id)
+            product_sizes      = product.productsizes.all()
+            product_urls       = [image.url for image in product.productimages.all()]
+            information_urls   = [information_image.url for information_image in product.informationimages.all()]
+            size_price         = [{
+                'size'  : product_size.size.size,
+                'price' : product_size.price
+                } for product_size in product.productsizes.all()]
```

Comment:
-  변수명 size_price - > sizes
- size_id 추가

```suggestion
            sizes         = [{
                'size_id' : product_size.size.id,
                'size'     : product_size.size.size,
                'price'    : product_size.price
            } for product_size in product.productsizes.all()]
```

Reply:
id를 추가했습니다.